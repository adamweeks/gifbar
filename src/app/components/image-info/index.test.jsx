import React from 'react';
import renderer from 'react-test-renderer';

import ImageInfo from '.';

describe(`ImageInfo component`, () => {
  const component = renderer.create(
    <ImageInfo
      info={`Image Info`}
    />
  );

  it(`renders properly`, () => {
    expect(component).toMatchSnapshot();
  });
});
