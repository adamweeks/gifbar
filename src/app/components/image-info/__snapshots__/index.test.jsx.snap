// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImageInfo component renders properly 1`] = `
<div
  className="info"
>
  Image Info
</div>
`;
